### ISSUE TITLE
Can't start with Java 1.7

### ISSUE THREAD

**Issue by inglor** [freeplane/freeplane#83]
Exception during start up

```
$ freeplane
org.knopflerfish.framework.readonly=true
org.knopflerfish.gosg.jars=reference:file:/usr/share/freeplane/core/
org.freeplane.user.dir=/home/zako
org.freeplane.basedirectory=/usr/share/freeplane
java.security.policy=/usr/share/freeplane/freeplane.policy
org.osgi.framework.storage=/usr/share/freeplane/fwdir
Knopflerfish OSGi framework launcher, version <unknown>
Copyright 2003-2017 Knopflerfish. All Rights Reserved.
See http://www.knopflerfish.org for more information.

Created Framework: org.knopflerfish.framework, version=8.0.4.
User properties not found, new file created
Installed: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.bugreport (id#2)
Installed: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.formula (id#3)
Installed: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.jsyntaxpane (id#4)
Installed: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.latex (id#5)
Installed: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.openmaps (id#6)
Installed: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.script (id#7)
Installed: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.svg (id#8)
Started: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.bugreport (id#2)
Started: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.formula (id#3)
Started: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.jsyntaxpane (id#4)
Started: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.latex (id#5)
Started: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.openmaps (id#6)
Started: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.script (id#7)
Started: reference:file:/usr/share/freeplane/plugins/org.freeplane.plugin.svg (id#8)
Oct 08, 2017 10:34:40 PM org.freeplane.core.util.LogUtils info
INFO: language resources for eu found
Oct 08, 2017 10:34:41 PM org.freeplane.core.util.LogUtils info
INFO: freeplane_version = 1.6.7; freeplane_xml_version = freeplane 1.6.0
git revision = a51dc03bcd0dc661aace9bfb7f83e7f66eccc133 Dimitry Polivaev 2017-09-23T14:40:22Z
java_version = 1.7.0_151; os_name = Linux; os_version = 4.13.4-1-ARCH

STDERR: java.lang.NoSuchMethodError: java.util.Properties.remove(Ljava/lang/Object;Ljava/lang/Object;)Z
STDERR: at org.freeplane.features.ui.FrameController.setLookAndFeel(FrameController.java:554)
STDERR: at org.freeplane.main.application.FreeplaneGUIStarter.createController(FreeplaneGUIStarter.java:175)
STDERR: at org.freeplane.main.osgi.ActivatorImpl.startFramework(ActivatorImpl.java:183)
STDERR: at org.freeplane.main.osgi.ActivatorImpl.start(ActivatorImpl.java:84)
STDERR: at org.freeplane.main.osgi.Activator.start(Activator.java:36)
STDERR: at org.knopflerfish.framework.BundleImpl.start0(BundleImpl.java:386)
STDERR: at org.knopflerfish.framework.BundleThread.run(BundleThread.java:145)
STDERR: ## DEBUG: errors - FrameworkErrorEvent bundle #1
STDERR: ## DEBUG: errors - FrameworkErrorEvent throwable:
STDERR: org.osgi.framework.BundleException: Bundle#1 start failed
STDERR: at org.knopflerfish.framework.BundleImpl.start0(BundleImpl.java:420)
STDERR: at org.knopflerfish.framework.BundleThread.run(BundleThread.java:145)
STDERR: Caused by: java.lang.NoSuchMethodError: java.util.Properties.remove(Ljava/lang/Object;Ljava/lang/Object;)Z
STDERR: at org.freeplane.features.ui.FrameController.setLookAndFeel(FrameController.java:554)
STDERR: at org.freeplane.main.application.FreeplaneGUIStarter.createController(FreeplaneGUIStarter.java:175)
STDERR: at org.freeplane.main.osgi.ActivatorImpl.startFramework(ActivatorImpl.java:183)
STDERR: at org.freeplane.main.osgi.ActivatorImpl.start(ActivatorImpl.java:84)
STDERR: at org.freeplane.main.osgi.Activator.start(Activator.java:36)
STDERR: at org.knopflerfish.framework.BundleImpl.start0(BundleImpl.java:386)
STDERR: ... 1 more
STDERR: Nested bundle exception:
STDERR: java.lang.NoSuchMethodError: java.util.Properties.remove(Ljava/lang/Object;Ljava/lang/Object;)Z
STDERR: at org.freeplane.features.ui.FrameController.setLookAndFeel(FrameController.java:554)
STDERR: at org.freeplane.main.application.FreeplaneGUIStarter.createController(FreeplaneGUIStarter.java:175)
STDERR: at org.freeplane.main.osgi.ActivatorImpl.startFramework(ActivatorImpl.java:183)
STDERR: at org.freeplane.main.osgi.ActivatorImpl.start(ActivatorImpl.java:84)
STDERR: at org.freeplane.main.osgi.Activator.start(Activator.java:36)
STDERR: at org.knopflerfish.framework.BundleImpl.start0(BundleImpl.java:386)
STDERR: at org.knopflerfish.framework.BundleThread.run(BundleThread.java:145)
```

**Comment by inglor** [comment]
Raised a PR to fix this #84

**Comment by fnatter** [comment]
Closing since PR is merged.
